terraform/modules/compute/wls_compute/variables.tf

```diff
@@ -26,6 +26,12 @@ variable "availability_domain" {
   description = "The label of the availability domain where the compute will be created"
 }
 
+variable "place_all_compute_in_same_ad" {
+  type        = bool
+  description = "Set to true if you want to use same Availability Domain for all Compute Instances."
```

Comment:
"to make a bit more clear - say 'AD of admin compute instance' "

Same comment for the description field in other .tf file.

Reply:
Modified the description